src/modules/peek/Peek.UI/MainWindow.xaml

```diff
@@ -17,11 +17,15 @@
 
     <Grid>
         <Grid.RowDefinitions>
-            <RowDefinition Height="32" />
+            <RowDefinition Height="48" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
 
-        <views:TitleBar x:Name="TitleBarControl" Grid.Row="0" />
+        <views:TitleBar
+            x:Name="TitleBarControl"
+            Grid.Row="0"
+            File="{x:Bind ViewModel.CurrentFile, Mode=OneWay}"
+            NumberOfFiles="{x:Bind ViewModel.Files.Count, Mode=OneWay}" />
```

Comment:
Sorry! My navigation/file query work that was merged in will probably conflict with this. You'll have to pull it from FolderItemsQuery.Files. Though I can see some issues, since we're now updating Files and CurrentItemIndex from a background thread so we might have some wrong thread exceptions if we make the properties observable. 

If it's too much work, it might be worth doing this dynamic count updating in a separate PR.